src/server/controllers/transactions.js

```diff
@@ -34,10 +34,7 @@ export async function invoice(req, res, next) {
     return next(e);
   }
 
-  const pageFormat =
-    req.query.pageFormat === 'A4' || invoice.fromCollective.currency === 'EUR'
-      ? 'A4'
-      : 'Letter';
+  const pageFormat ='A4';
```

Comment:
Why?

Format should be different for Europe (A4) and US (Letter).

Reply:
Hum, I see. the Letter format is going over the pdf width limit, looks buggy to me but I'll double check and look for a better workaround.